project/Build.scala

```diff
@@ -409,15 +408,8 @@ object Build {
       pomIncludeRepository := { _ => false }
   )
 
-  val fatalWarningsSettings = Seq(
-      // The pattern matcher used to exceed its analysis budget before 2.11.5
-      scalacOptions ++= {
-        scalaVersion.value.split('.') match {
-          case Array("2", "11", x)
-              if x.takeWhile(_.isDigit).toInt <= 4 => Nil
-          case _                                   => Seq("-Xfatal-warnings")
-        }
-      },
+  val fatalWarningsSettings = Def.settings(
+      scalacOptions += "-Xfatal-warnings"
```

Comment:
Should we inline this? Seems a bit pointless like this

Reply:
Well, I think it's still relevant to abstract it away, if anything so that we don't have to copy-paste a string literal, which is not type-checked.

Reply:
OK. Fair enough.